intake_dal/dal_online.py

```diff
@@ -25,6 +26,8 @@ class DalOnlineSource(DataSource):
     name = "dal-online"
     version = pkg_resources.get_distribution("intake-dal").version
 
+    __date_time_format = "%Y-%m-%d %H:%M:%S.%f"
```

Comment:
Let's make this a constant with uppercase like `GLOBAL_CONSTANT_NAME` in http://google.github.io/styleguide/pyguide.html

Reply:
done